src/routes/reference/jsx-attributes/bool.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: bool:*
+---
+
+It controls the presence of an attribute in an element.
+When the value is `truthy` it adds the `attribute` to the element.
+When the value is `falsy` it removes the `attribute` from the element.
+Mostly useful for Web Components.
+
+```tsx
+<my-element bool:status={prop.value} />
+```
+
+Assuming `prop.value` is `truthy`, then it becomes
+
+```tsx
+<my-element status />
+```
+
+And when `falsy`
+
+```tsx
+<my-element bool:status={""} />
+```
+
+Becomes
+
+```tsx
+<my-element />
+```
```

Comment:
The snippet on top is the reference, and the others are the examples, I would prefer to not mix the reference with the examples. The page is small enough and have space, separated communicates the idea clearly and better than in a chunk of code imo.

Reply:
In keeping them together, then, rather than giving them entire code blocks I would much rather they remain inline.

It's not a matter of space but readability. Having them all in code blocks is a bit excessive for one line of code.

Reply:
ok, there its on one block of code, let me know if that's better

Reply:
Thank you! I'll keep this on hold for when/if the future lands